src/portable/portable.c

```diff
@@ -994,16 +1058,29 @@ static int install_chroot_dropin(
                                "LogExtraFields=PORTABLE=", base_name, "\n"))
                         return -ENOMEM;
 
-                if (!ordered_hashmap_isempty(extension_images)) {
+                /* If we have a single image then PORTABLE= will point to it, so we add PORTABLE_ID= with
+                 * the os-release fields and we are done. But if we have extensions, PORTABLE= will point
+                 * to the image where the current unit was found in. So we also list PORTABLE_ROOT= and
+                 * PORTABLE_ROOT_ID= for the base image, and PORTABLE_EXTENSION= and PORTABLE_EXTENSION_ID=
+                 * for each extension, so that all needed metadata is available. */
+                if (ordered_hashmap_isempty(extension_images)) {
+                        r = append_release_log_fields(os_release, IMAGE_PORTABLE, "PORTABLE_ID", &text);
+                } else {
```

Comment:
no {} around single-line if blocks,as per coding style

Reply:
hmm, the combination of a basename and a version should this really be called "ID"? rpm calls this "nevra" (i.e. name, epoch, version, release, architecture). Maybe we should call this something like "object"? or be literal, i.e. "PORTABLE_NAME_AND_VERSION"? PORTABLE_NAME_VERSION, PORTABLE_NVER? PORTABLE_NAMVER? or something like this?

so far in /etc/os-release we used the "_ID" suffix a bit chaotic but usually it's used as a low-level identifier rather than something to expose to the user. it's never a combination of things. This here seems to be quite a different use though, hence we should really pick a different moniker.

PORTABLE_MONIKER? PORTABLE_NAME?

kinda like "moniker" for this...